pages/common/ogr2ogr.md

```diff
@@ -0,0 +1,24 @@
+# ogr2ogr
+
+> Convert Simple Features data between file formats.
+> More information: <https://gdal.org/programs/ogr2ogr.html#ogr2ogr>.
+
+- Convert a Shapefile into a GeoPackage:
+
+`ogr2ogr -f GPKG {{output}}.gpkg {{input}}.shp`
+
+- Reproject a GeoPackage from `EPSG:4326` to `EPSG:3857` coordinate reference system:
```

Comment:
```suggestion
- Reproject a GeoPackage changing coordinate reference system from `EPSG:4326` to `EPSG:3857`:
```

Reply:
I changed the wording to:

`- Change coordinate reference system of a GeoPackage from `EPSG:4326` to `EPSG:3857``

Reply:
@mebeim do you agree on my changed wording?

Reply:
Oh, sorry I missed this comment, yes it looks fine to me, good job :)